android-components/components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml

```diff
@@ -132,6 +132,31 @@
                 android:text="@string/mozac_feature_addons_status_learn_more" />
         </LinearLayout>
 
+        <LinearLayout
+            android:id="@+id/add_on_status_unsigned"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:orientation="vertical"
+            android:visibility="gone">
+
+            <TextView
+                android:id="@+id/add_on_status_unsigned_message"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:textSize="14sp"
+                android:textColor="@color/mozac_feature_addons_error_text_color"
+                tools:text="@string/mozac_feature_addons_status_unsigned" />
+
+            <TextView
+                android:id="@+id/add_on_status_unsigned_learn_more_link"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:textSize="14sp"
+                android:textColor="?android:attr/textColorLink"
+                android:text="@string/mozac_feature_addons_status_learn_more" />
+        </LinearLayout>
```

Comment:
As the structure for **blocklisted** looks really similar, couldn't we just reuse the previous UI?